README.md

```diff
@@ -244,3 +244,30 @@ jupyterhub::jupyter_notebook_config_hash:
 | `jupyterhub::oauthenticator::scope` | Array[String] | OIDC scope | |
 | `jupyterhub::oauthenticator::allowed_groups` | Array[String] | List of groups who should be allowed to connect. Empty list = any group | [] |
 | `jupyterhub::oauthenticator::claim_groups_key` | String | Userdata groups claim key from returned  OIDC json | 'affiliation' |
+
+### Submit addition option
+| Variable | Type | Description |
+| -------- | :----| :-----------|
+| `jupyterhub::submit::additions` | String | Script that should be added to submit.sh |
```

Comment:
Perhaps change wording to indicate that it is "bash command(s)" and not "Script"

Reply:
done